tests/python/test_grammar_matcher_ebnf.py

```diff
@@ -352,7 +520,12 @@ def test_fill_next_token_bitmask(
     print(f"Time to init GrammarMatcher: {(time_end - time_start) / 1e3} us")
 
     token_bitmask = xgr.allocate_token_bitmask(1, tokenizer_info.vocab_size)
-    logits_gpu = torch.zeros(tokenizer_info.vocab_size, dtype=torch.float32, device="cuda")
+    device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
these tests will fallback to a CPU path if CUDA isn't available now which is nice for MacOS